example/mailto/rfc.hpp

```diff
@@ -139,13 +139,29 @@ constexpr auto atext_token =
     grammar::token_rule(atext_chars);
 
 /// Rule for dot-atom-text = 1*atext *("." 1*atext)
-constexpr auto dot_atom_text_rule =
-    grammar::range_rule(
-        atext_token,
-        grammar::tuple_rule(
-            grammar::squelch(
-                grammar::delim_rule('.')),
-            atext_token));
+struct dot_atom_text_rule_t {
+    using value_type = urls::string_view;
+
+    urls::result< value_type >
+    parse(
+        char const*& it,
+        char const* end) const noexcept
+    {
+        static auto r = grammar::range_rule(
+            atext_token,
+            grammar::tuple_rule(
+                grammar::squelch(
+                    grammar::delim_rule('.')),
+                atext_token));
```

Comment:
I would like to know what the compiler is emitting for the double-checked locking, if any

Reply:
Are there any good tools to do that locally? I'm only used to doing that with compiler explorer for small snippets but this PR is not going to be available there.

Reply:
I just set a breakpoint in visual studio and examine the assembly